recipes/default.rb

```diff
@@ -49,6 +49,12 @@
   conf['DEFAULT']['notify_on_state_change'] = 'vm_and_task_state'
   conf['DEFAULT']['disk_allocation_ratio'] = 1.5
   conf['libvirt']['disk_cachemodes'] = 'file=writeback,block=none'
+  conf['DEFAULT']['allow_resize_to_same_host'] = 'True'
+  conf['DEFAULT']['scheduler_default_filters'] = 'RetryFilter,
+  AvailabilityZoneFilter, RamFilter, DiskFilter, ComputeFilter,
+  ComputeCapabilitiesFilter, ImagePropertiesFilter,
+  ServerGroupAntiAffinityFilter, ServerGroupAffinityFilter,
+  AllHostsFilter'
```

Comment:
I'd change this to an array and use ``.join(',')`` at the end. That way it's easier to look at visually. Something like this:

``` ruby
 conf['DEFAULT']['scheduler_default_filters'] = %w(
  RetryFilter
￼  AvailabilityZoneFilter
  RamFilter
  DiskFilter
  ComputeFilter
￼  ComputeCapabilitiesFilter
  ImagePropertiesFilter
￼  ServerGroupAntiAffinityFilter
  ServerGroupAffinityFilter
￼  AllHostsFilter
).join(',')
```